upgrade_tests/upgrade_manifest.py

```diff
@@ -248,14 +248,16 @@ def build_upgrade_pairs():
                     logger.debug("{} appears applicable to current env. Overriding final test version from {} to {}".format(path_name, oldmeta.version, newmeta.version))
                     destination_meta = newmeta
 
-            valid_upgrade_pairs.append(
-                UpgradePath(
-                    name=path_name,
-                    starting_version=origin_meta.version,
-                    upgrade_version=destination_meta.version,
-                    starting_meta=origin_meta,
-                    upgrade_meta=destination_meta
+            from upgrade_tests.upgrade_through_versions_test import jdk_compatible_steps
```

Comment:
nit: maybe `jdk_compatible_steps` should be moved here ( to upgrade_manifest.py ).  this is more the base file.